FirebaseVertexAI/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.2.0
+
+- [added] Initial VertexAI release.
```

Comment:
Can we add a link to the announcement and detailed docs?

Reply:
Announcement TBD, but I added a link to the docs.